src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -82,4 +100,36 @@ public Command parseCommand(String userInput) throws ParseException {
         }
     }
 
+    /**
+     * Checks if command is a substring of a full command.
+     *
+     * @param command user input command string
+     * @return  full command if input is a substring, else return the original input command
+     * @throws ParseException if the input matches to multiple commands
+     */
+    private String fullCommand(String command) throws ParseException {
```

Comment:
>  Names representing methods **must be verbs** and written in camelCase.

https://se-education.org/guides/conventions/java/intermediate.html

Reply:
is fullCommand not in camelCasing?

Reply:
Important part is "must be verbs".